x-pack/legacy/plugins/siem/public/components/toasters/errors.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export class ToasterError extends Error {
+  public messages: string[];
```

Comment:
I'd do this:

```ts
public readonly messages: string[];
```

So you don't have to worry about people mutating this array.